api/v1/services/index.js

```diff
@@ -2,10 +2,10 @@ module.exports = {
 	AuthService: require('./AuthService'),
 	EcosystemService: require('./EcosystemService'),
 	MailService: require('./MailService'),
+	RegistrationService: require('./RegistrationService'),
```

Comment:
The registration functionality? I think it makes sense to go in its own service since there's nothing specifically related to projects beyond the fact that there are projects involved in registration.